documentation/descriptionreport.xml

```diff
@@ -0,0 +1,30 @@
+<project name="documentation" default="creatreport">
+
+	<target name="copyfiles">
+		<property name="src.dir" value="${basedir}/.." />
+		<property name="work.dir" value="${project.build.directory}/generated/${infinispan.base.version}/report/tmp" />
+		<property name="search.string" value="DescriptionDocumentation" />
+		<delete dir="${work.dir}" />
+		<mkdir dir="${work.dir}" />
+		<copy todir="${work.dir}" failonerror="false" >
+			<fileset dir="${src.dir}" defaultexcludes="true">
+				<include name="**/*.xml" />
+				<exclude name="**/descriptionreport.xml" />
+				<contains text="${search.string}" casesensitive="yes" />
+			</fileset>
+		</copy>
+	</target>
+	<target name="creatreport" depends="copyfiles">
+		<property name="temp.dir" value="${project.build.directory}/generated/${infinispan.base.version}/report/tmp" />
+		<property name="dest.dir" value="${project.build.directory}/generated/${infinispan.base.version}/report" />
```

Comment:
I think this should be `${project.build.directory}/generated/${infinispan.base.version}/logging`